blueprintapi/settings.py

```diff
@@ -201,10 +197,16 @@
 REST_FRAMEWORK = {
     "DEFAULT_AUTHENTICATION_CLASSES": [
         "blueprintapi.authentication.ExpiringTokenAuthentication",
+        "rest_framework.authentication.SessionAuthentication",
     ],
     "DEFAULT_PERMISSION_CLASSES": [
         "blueprintapi.permissions.StrictDjangoObjectPermissions",
     ],
     "DEFAULT_SCHEMA_CLASS": "rest_framework.schemas.coreapi.AutoSchema",
     "PAGE_SIZE": 20,
 }
+
+SWAGGER_SETTINGS = {
```

Comment:
Is this being referenced somewhere or is there some magic that happens behind the scenes?

Reply:
More or less magic. Similar to how you can set global settings with rest framework, `drf-yasg` also has some global settings that you can configure.